modules/apps/29-fee/types/events.go

```diff
@@ -1,4 +1,6 @@
 package types
 
 // 29-fee events
-const ()
+const (
+	EventTypeSendIncentivizedPacket = "send_incentivized_packet"
```

Comment:
The packet is already sent, the packet is just now being incentivized (or further incentivized)